### ISSUE TITLE
withFileFromClassPath fails when path to classpath resource contains spaces

### ISSUE THREAD

**Issue by lbialy** [testcontainers/testcontainers-java#263]
Hello,

How to replicate:
1. create directory with spaces in path
```bash
mkdir ~/dir\ with\ spaces
```
2. move some project using testcontainers-java to that directory
3. put a file to copy into container in `resources` directory (`touch src/test/resources/any_file` in this example)
4. use container loading files from class path, ie:
```java
new GenericContainer(
        new ImageFromDockerfile()
                .withDockerfileFromBuilder(builder -> {
                        builder
                                .from("alpine:3.2")
                                .copy("any_file", "any_file")
                                .build(); 
                        })
                )
                .withFileFromClasspath("any_file", "any_file")
)
```

This will throw exception:
```
1 error
Caused by: org.testcontainers.containers.ContainerFetchException: Can't get Docker image name from org.testcontainers.images.builder.ImageFromDockerfile@61ae0d43
Caused by: java.lang.RuntimeException: Can't get size from ~/dir%20with%20spaces/your-project/target/test-classes/any_file
Caused by: java.nio.file.NoSuchFileException: ~/dir%20with%20spaces/your-project/target/test-classes/any_file
```

Issue is related to urlencoded spaces in path (%20) when using 
```
URL resource = ClasspathTrait.class.getClassLoader().getResource(resourcePath);
String resourceFilePath = new File(resource.getFile()).getAbsolutePath();
```
in default implementation of `withFileFromClasspath`. Calling `.toURI()` instead of `.getFile()` on resource should fix this issue while not breaking anything - I'm using this workaround and seems to be fine. No PR though, sorry, simply no time to do it.

**Comment by rnorth** [comment]
Hi @lbialy 
Thanks for reporting, and sorry for the problem and thorough analysis. We'll try and address soon.
Richard

**Comment by lbialy** [comment]
@rnorth I think you wanted to mention #262, not #263 here. Glad to see something found.

**Comment by rnorth** [comment]
@lbialy this is hopefully fixed now - please could you let me know if you encounter any more issues? 

I've been developing under a path containing space for a while, just to make sure 😃

**Comment by bsideup** [comment]
Hi @lbialy
We released 1.3.0 with a few fixes to file mappings, could you please try it?
Thanks!

**Comment by lbialy** [comment]
I'll try, thanks!

**Comment by bsideup** [comment]
Fixed by #279.
